test/hotspot/jtreg/compiler/c2/irTests/TestVectorConditionalMove.java

```diff
@@ -46,7 +46,7 @@ public static void main(String[] args) {
         TestFramework.runWithFlags("-XX:-TieredCompilation",
                                    "-XX:+UseCMoveUnconditionally",
                                    "-XX:+UseVectorCmov",
-                                   "-XX:CompileCommand=compileonly,*.TestVectorConditionalMove.test*");
+                                   "-XX:CompileCommand=compileonly,compiler.c2.irTests.TestVectorConditionalMove::test*");
```

Comment:
`compileonly` is probably not necessary but you can leave it as such.

Reply:
Also removed the `-XX:-TieredCompilation` :)